parts/chapters/2.fodt

```diff
@@ -91748,13 +91747,13 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
       <table:table-row table:style-name="Table2273.1">
        <table:table-cell table:style-name="Table2273.A1" office:value-type="string">
         <text:p text:style-name="P603"/>
-        <text:p text:style-name="P603">Warning: LIQ preferred phase not supported for well OP-01, using OIL instead</text:p>
+        <text:p text:style-name="P603">Warning: LIQ preferred phase not supported for well OP-01, using <text:a xlink:href="#__RefHeading___Toc97439_1778172979">OIL</text:a> instead</text:p>
```

Comment:
I think we need to guard this one. It should not be a link.

Reply:
Good catch. Can we check for style P603 here? It seem like both cases uses that style to produce a mono space font on a gray background

Reply:
Like Matthew I would assume that these numbers might change.

Proably, having an extra link is not such a problem. Hence we should not invest too much time.